docs/how-to-guides/index.md

```diff
@@ -17,6 +17,14 @@ chat-with-a-flow/index
 run-and-evaluate-a-flow/index
 ```
 
+```{toctree}
+:caption: Prompty
+:maxdepth: 1
+develop-a-prompty/index
+develop-a-prompty/prompty-output-format
```

Comment:
no need to have these two as index covers them?

Reply:
I think it's better to keep these two doc. The prompt structure and execution method are briefly introduced in index.
The other two documents introduce the more complex configuration methods of prompt and the usage of nesting into other types of flows.